spec/factories/foods.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :food do
+    sequence(:name) { |n| "Food #{n}" }
+    unit_name 'g'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.